source/leuven/tier2_hardware/wice_hardware.rst

```diff
@@ -44,3 +44,41 @@ The nodes are connected using an Infiniband HDR-100 network, the islands are ind
 
 .. figure:: wice_hardware/wice.png
    :alt: wICE hardware diagram
+
+
+Hardware details (extension)
+----------------------------
+We are currently installing and testing additional hardware for wICE,
+which will be made accessible during the spring of 2024:
+
+- 68 thin nodes
+
+  - 2 Intel Xeon Platinum 8468 CPUs (Sapphire Rapids),
+    48 cores each :raw-html:`<br />`
+    The base and max CPU frequencies are 2.1 GHz and 3.8 GHz, respectively.
+  - 256 GB RAM
+  - 960 GB SSD local disk
+  - partitions ``batch_sapphirerapids/batch_sapphirerapids_long``
+
+- 4 GPU nodes, 16 GPU devices
+
+  - 2 AMD EPYC 9334 CPUs (Genoa),
+    32 cores each :raw-html:`<br />`
+    The base and max CPU frequencies are 2.7 GHz and 3.9 GHz, respectively.
+  - 770 GB RAM
```

Comment:
Slightly pedantic, but should this not be 768 GiB (=24 * 32 GiB)? Not entirely sure about the GiB vs GB.

Reply:
I was basing it on `lsmem` output, but it indeed looks like that cannot always be thrusted (conflicting with what you'd get by counting the DIMMs and with other commands like `lshw`). And all these numbers should be GiB, not GB. I fixed this one and also the entries in our other hardware detail pages (41cf442b6f1d814bd96c65b769dc911d174f1baa).

(Side note: `nvidia-smi` actually reports 79.65 GiB and not 80 GiB for our H100 nodes, but so I've rounded this up to 80 GiB)